packages/components/src/tree-grid/row.tsx

```diff
@@ -28,7 +29,7 @@ function UnforwardedTreeGridRow(
 			aria-level={ level }
 			aria-posinset={ positionInSet }
 			aria-setsize={ setSize }
-			aria-expanded={ isExpanded }
```

Comment:
This is the best area to explain this. The reason why this needs to exist is as follows.
1. If aria-expanded exists on the tr and the link, it is announced twice when it is toggled.
2. If it only exists on the row, it is announced when toggled but not after that.
3. If it exists on the link only, it works fine when toggled and after.
4. Had to figure out how to eliminate it off the tr.
5. The better way about this may be to add data-expanded directly here and pass the value from the `ListView`. Something like this.
```
aria-expanded={ disableAriaExpanded ? undefined : isExpanded }
data-expanded={ disableAriaExpanded ? isExpanded : undefined }
```
One of these attributes is required for the keyboard functionality. Not pretty, but it works.